src/dune_pkg/import.ml

```diff
@@ -0,0 +1,4 @@
+include Stdune
+open Dune_util
```

Comment:
We probably don't need to do this. Having the module qualified makes it easier to track what is coming from where.